ca/ca.go

```diff
@@ -601,3 +565,39 @@ func (ca *CertificateAuthorityImpl) issueCertificateOrPrecertificate(ctx context
 
 	return certDER, nil
 }
+
+func (ca *CertificateAuthorityImpl) generateOCSPAndStoreCertificate(ctx context.Context, regID int64, serialBigInt *big.Int, certDER []byte) (core.Certificate, error) {
+	var ocspResp []byte
+	var err error
+	if features.Enabled(features.GenerateOCSPEarly) {
+		ocspResp, err = ca.GenerateOCSP(ctx, core.OCSPSigningRequest{
+			CertDER: certDER,
+			Status:  "good",
+		})
+		if err != nil {
+			err = berrors.InternalServerError(err.Error())
+			ca.log.AuditInfo(fmt.Sprintf("OCSP Signing failure: serial=[%s] err=[%s]", core.SerialToString(serialBigInt), err))
+			// Ignore errors here to avoid orphaning the certificate. The
+			// ocsp-updater will look for certs with a zero ocspLastUpdated
+			// and generate the initial response in this case.
+		}
+	}
+
+	// Store the cert with the certificate authority, if provided
```

Comment:
I see that this comment was brought over from the original code unmodified. I'm not sure I understand it. It seems like the SA is provided unconditionally and there is always a call to `ca.sa.AddCertificate`. Can you remove this comment or reword to make it clearer if I've misunderstood?